addons/medical/stringtable.xml

```diff
@@ -42,14 +42,7 @@
             <Spanish>La probabilidad de que una unidad estabilizada recupere la conciencia y se levante (se comprueba cada 15 segundos)</Spanish>
         </Key>
         <Key ID="STR_ACE_Medical_spontaneousWakeUpEpinephrineBoost_DisplayName">
-            <English>Epinephrine Increases Wake Up Chance</English>
-            <Japanese>アドレナリンで覚醒確率上昇</Japanese>
-            <Chinese>腎上腺素甦醒機率增加</Chinese>
-            <French>L'épinéphrine augmente les chances de réveil</French>
-            <Italian>L'epinefrina aumenta la possibilità di rialzarsi</Italian>
-            <Czech>Epinefrin zvyšuje šanci na probuzení</Czech>
-            <Portuguese>Epinefrina Aumenta a chance de acordar</Portuguese>
-            <Spanish>Epinefrina, aumenta la posibilidad de despertar</Spanish>
+            <English>Epinephrine Wake Up Chance Boost</English>
```

Comment:
Boosts wake up chance?

Reply:
This is a string for a setting name, it's not an explanation of what it does.

Reply:
ref https://github.com/acemod/ACE3/blob/master/addons/medical_statemachine/functions/fnc_handleStateUnconscious.sqf#L37-L42

It doesn't directly change the wake up chance, 
but instead decreases the time between doing wakeup random
I did it this way because epi is somewhat short lived, so I felt like this would be the most responsive

Not positive on the math, but the effect should be similar

Reply:
Yeah, math looks good, but the setting name was confusing.